images/win/scripts/Tests/Databases.Tests.ps1

```diff
@@ -53,8 +53,10 @@ Describe "PostgreSQL" {
 }
 
 Describe "MySQL" {
-    It "MySQL CLI" {
-        "mysql -V" | Should -ReturnZeroExitCode
-    }
-}
+    It "MySQL version should correspond to the version in the toolset" {
+        $MysqlShortVersion = (Get-ToolsetContent).Mysql.short_version
+        $MysqlFullVersion = ((Invoke-WebRequest -Uri https://dev.mysql.com/downloads/mysql/${MysqlShortVersion}.html).Content | Select-String -Pattern "${MysqlShortVersion}\.\d+").Matches.Value
 
+        mysql --version | Should -BeLike "*${MysqlFullVersion}*"
```

Comment:
I don't think that we need to check the exact version here. It is enough to check whether `mysql --version` is like `MysqlMajorMinor` or not.

Reply:
I fixed it. Thanks.

Reply:
Looks like no

Reply:
Deleted it. Thanks.